ximea_ros2_cam/include/ximea_ros2_cam/ximea_ros2_cam.hpp

```diff
@@ -89,6 +89,8 @@ class XimeaROSCam : public rclcpp::Node, std::enable_shared_from_this<XimeaROSCa
    static std::map<std::string, std::string> ImgEncodingMap;
    static std::map<int, int> CamMaxPixelWidth;
    static std::map<int, int> CamMaxPixelHeight;
+   static std::map<std::string, int> DownsamplingMap;
```

Comment:
document each attribute

Reply:
done

Reply:
Commit: https://github.com/Brazilian-Institute-of-Robotics/ximea_ROS2_driver/pull/1/commits/2bb6e0eae9117872aa357f1acf15b0cc064acba2